examples/pom.xml

```diff
@@ -35,6 +35,10 @@
     <sbt.project.name>examples</sbt.project.name>
     <build.testJarPhase>none</build.testJarPhase>
     <build.copyDependenciesPhase>package</build.copyDependenciesPhase>
+    <flume.deps.scope>provided</flume.deps.scope>
```

Comment:
Yeah, these dependencies are explicitly declared with `compile` scope in the root pom, and these properties override that.